wgsl/index.bs

```diff
@@ -16878,6 +16919,62 @@ Note: For packing snorm values, the normalized floating point values are in the
         8 &times; `i` + 7 of the result.
 </table>
 
+### `pack4xI8` ### {#pack4xI8-builtin}
+<table class='data builtin'>
```

Comment:
The pack and unpack built-in function tables are missing the `Overload` entry so the table style doesn't match other built-ins.